hubcast/__main__.py

```diff
@@ -46,8 +52,52 @@ async def handle(self, request: web.Request) -> Any:
         return web.Response(status=404)
 
 
+def setup_logging():
+    # Standard Logging file
+    config = {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "default": {
+                "format": "[%(levelname)s|%(module)s|L%(lineno)d] %(asctime)s: %(message)s",
+                "datefmt": "%Y-%m-%dT%H:%M:%S%z",
+            },
+            "json": {
+                "()": "hubcast.logging.json_logging.MyJSONFormatter",
+                "fmt_keys": {
+                    "level": "levelname",
+                    "message": "message",
+                    "timestamp": "timestamp",
+                    "logger": "name",
+                    "module": "module",
+                    "function": "funcName",
+                    "line": "lineno",
+                },
+            },
+        },
+        "handlers": {
+            "stdout": {
+                "class": "logging.StreamHandler",
+                "formatter": "json",
+                "stream": "ext://sys.stdout",
+            }
+        },
+        "loggers": {"root": {"level": "DEBUG", "handlers": ["stdout"]}},
+    }
+
+    # Custom logging file
+    config_file = pathlib.Path("logging/local_logging_config.json")
+    if config_file.exists():
+        with open(config_file) as f_in:
+            config = json.load(f_in)
```

Comment:
Instead of hardcoding a value let's derive the location of the logging configuration from an optional config value. Perhaps `HC_LOGGING_CONFIG_PATH`?

Reply:
sounds good i modified it